style.css

```diff
@@ -13,10 +14,21 @@ body > * {
 }
 
 .header {
-  background: darkgrey;
+  /* background: darkgrey; */
   height: 50px;
   font-size: 25px;
-  margin: 0;
+  margin-bottom: 0%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 0 2rem;
+}
+.logo {
+  height: 45px;
+}
+
+.footer {
+  margin-top: auto;
```

Comment:
what is it for?

Reply:
It's for position footer at the very bottom of page